README.md

```diff
@@ -297,26 +297,20 @@ The following steps describe the minimal steps required to successfully log in t
 1. If using Visual Studio Code, you may need to set `FAST_REFRESH=false` in your .env file
 1. Azure Data Studio is an adequate program for manipulating the SQL Server database
 
-### Admin screens/routes
+### UID2 Support Screens/Routes
```

Comment:
This renaming breaks the link on [line 290](https://github.com/IABTechLab/uid2-self-serve-portal/blob/main/README.md?plain=1#L290).